src/Sylius/Behat/Context/Ui/CheckoutContext.php

```diff
@@ -200,6 +229,7 @@ public function iProceedLoggingAsGuestWithAsShippingCountry($email, $shippingCou
      */
     public function iConfirmMyOrder()
     {
+        $this->paypalApiMocker->mockApiPaymentInitializeResponse();
```

Comment:
We will mock this even if PayPal is not used? :/ Seems like we need to have PayPalCheckoutContext that is specific to PayPal implementation... :/